byron/ledger/formal-spec/blockchain-interface.tex

```diff
@@ -446,6 +446,73 @@ \subsection{Update-proposals interface}
 
 \clearpage
 
+A sequence of votes can be applied using $\trans{upivotes}{}$ transitions. The
```

Comment:
How about adding this STS's type to "Figure 36: Update-proposals interface transition-system types"?

Reply:
Here I chose to omit the types since they are the same as in `UPIVOTE`.